src/components/InstanceSwitcher/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+/**
+ * Only purpose is to cause the website to refresh when switching between "instances",
+ * as a normal redirect from, say, `/platform/actors/tasks` to `/api/client/js/..` will not work by default (since they are
+ * different instances of Docusaurus)
```

Comment:
can we use something from this instead?

https://docusaurus.io/docs/advanced/routing#escaping-from-spa-redirects

Reply:
Ohhh, I completely missed that, let me try, if that works thats a MUCH cleaner solution

Reply:
It does work, but it opens the link in a new tab, which is not ideal

Reply:
Dang, we don't want that, but we could probably swizzle the Link component and adjust how that works, I already did that for the nav links.

https://github.com/apify/apify-docs-v2/blob/master/apify-docs-theme/src/theme/NavbarItem/NavbarNavLink.jsx#L32-L37

Reply:
I forgot that's a possibility already 😮‍💨. Pushed!